topics/wordplate/index.md

```diff
@@ -0,0 +1,11 @@
+---
+topic: wordplate
+created_by: Vincent Klaiber
+display_name: WordPlate
+github_url: https://github.com/wordplate
+logo: wordplate.png
+released: March 18, 2015
```

Comment:
You could add a `related: wordpress, composer, laravel, symfony, php` line to give more context.

Reply:
Done!